ansible_collections/arista/avd/molecule/evpn_underlay_ebgp_overlay_ebgp/intended/structured_configs/DC1-LEAF1A.yml

```diff
@@ -177,6 +177,7 @@ ethernet_interfaces:
     vlans: 1-4094
     spanning_tree_portfast: edge
     spanning_tree_bpdufilter: true
+    spanning_tree_bpduguard: false
```

Comment:
I don't see an update of the group_var files in this molecule testcase, so how are these changes implemented? The structured_configs are part of the output of the molecule, so they should not be changed directly.
You should change the "DC1_SERVERS.yml" group_var file to include this new knob, and then run the molecule scenarios using "make refresh-facts" from the molecule main folder.

Reply:
I noticed that too, but if you look in the DC1_SERVERS.yml you can see that someone had spanning_tree_bpduguard added before this to multiple parts.

Reply:
Haha good point. Then please just run the "make refresh-facts" so all molecule test cases run with your changes, and commit the resulting changes. Then you should be good.
Other than that, everything looks good to me!